lib/repl.js

```diff
@@ -226,7 +226,7 @@ function REPLServer(prompt,
       }
     }
 
-    if (!wrappedCmd && experimentalModules && code.startsWith('import')) {
+    if (!wrappedCmd && experimentalModules && /^(\s+?)?import/.test(code)) {
```

Comment:
I could see the sniff getting pretty robust... what about leading multiline comments, e.g. `/*x*/ import...`
Since `processTopLevelImport` fails gracefully you might just keep the check super simple `/\bimport\b.test(code)`

Reply:
I'll just change it to code.includes('import') at this point

Reply:
Cool, the word boundary check just prevents "important", "imports", etc.